internal_ws/ykcompile/src/lib.rs

```diff
@@ -1371,7 +1383,11 @@ impl TraceCompiler {
         // saved registers, when calculating the alignment.
         // While we could use the stack builder to keep track of the stack and the alignment, this
         // would require us to split up various dynasm! blocks, which is inconvenient.
-        let topalign = 16 - (live_off + soff as usize + CALLEE_SAVED_REGS.len() * 8) % 16;
+        //
+        // FIXME Lukas, what's the `16 - ` for here again?
```

Comment:
The `16 -` should be `SYSV_CALL_STACK_ALIGN -` I think. If for example `live_off + soff as usize + CALLEE_SAVED_REGS.len() * 8` is 23, `% 16` you get 7. To re-align the stack, you need 16 - 7 = 9 bytes extra.

Reply:
Makes sense. Let's confirm with Lukas (he's AFK today though).

Reply:
What @bjorn3 said.